composer/core/types.py

```diff
@@ -17,7 +17,7 @@
 
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Dict, Iterator, List, Optional, Sequence, Union
+from typing import TYPE_CHECKING, Dict, Iterator, List, Optional, Union
```

Comment:
```suggestion
from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Optional, Union
```

Reply:
done in [acf116c](https://github.com/mosaicml/composer/pull/954/commits/acf116c968f0458ddffd28ab9a4ad25b98c7b182)